docs/configuration.rst

```diff
@@ -2691,6 +2691,13 @@ Default
 Description
     Fetch media from quoted Tweets.
 
+    If the value is ``true``, then gallery-dl will try to fetch
+    the quoted (original) Tweet when it sees the Tweet which quotes it.
```

Comment:
Technically this is not what really happens, since Twitter's API returns both the quoting and the quoted tweet, and gallery-dl simply discards or accepts the quoted tweet depending on `quoted`. But I think it's easier to understand.